.babelrc

```diff
@@ -1,17 +1,30 @@
 {
   "presets": [
-    "flow",
-    "react",
-    ["env", {"modules": false}],
-    "stage-0"
+    "@babel/flow",
+    "@babel/react",
+    ["@babel/env", {"modules": false}]
   ],
   "plugins": [
-    ["relay", {"compat": true, "schema": "app/graph/schema.json"}]
+    ["relay", {"compat": true, "schema": "app/graph/schema.json"}],
```

Comment:
Won’t need this anymore!

Reply:
Oh, really? Can it just be dropped?

Reply:
Oh wait — sorry, I was thinking about webpack when I wrote this lol. Excuse me, I haven’t had a coffee yet 🙄